packages/chatbot-server-mongodb-public/src/processors/makeStepBackRagGenerateUserPrompt.ts

```diff
@@ -137,8 +136,12 @@ export const makeStepBackRagGenerateUserPrompt = ({
         url: c.url,
         score: c.score,
       })),
-      customData,
-      preprocessedContent: transformedUserQuery,
```

Comment:
To be clear - we will no longer be using the `preprocessedContent` field in user messages? Do you know if anything relies on this field, e.g. in `scripts` or some other package?

Reply:
the only place it exists in the `scripts` package is within `Omit<T, "preprocessedContent"` type defs, so i think safe to remove. no other meaningful usage elsewhere